site/content/docs/main/config/request-routing.md

```diff
@@ -105,8 +105,13 @@ Regex conditions **must** start with a `/` if they are present.
 
 #### Header conditions
 
-For `header` conditions there is one required field, `name`, six operator fields: `present`, `notpresent`, `contains`, `notcontains`, `exact`, and `notexact` and a modifier `ignoreCase` which can be used together with all of the operator fields except `regex`, `notpresent` and `present`.
+For `header` conditions there is the following structure:
 
+1. one required field, `name`
+2. six operator fields: `present`, `notpresent`, `contains`, `notcontains`, `exact`, and `notexact`
+3. two optional modifiers: `ignoreCase` and `TreatMissingAsEmpty`
```

Comment:
```suggestion
3. two optional modifiers: `ignoreCase` and `treatMissingAsEmpty`
```

Reply:
fixed in bdd8b20a58135f00906b651196b6240e56dba593